addons/stock/static/src/components/reception_report_main/stock_reception_report_main.js

```diff
@@ -128,12 +128,21 @@ export class ReceptionReportMain extends Component {
         return this.props.action.context;
     }
 
+    /**
+     * Context passed `default_{model}_ids` variables value depends on if its the stock or mrp use case
+     *
+     * @return {Array<String, Array>} list of 2 values: "default_{model}_ids", docIds
+     */
+    get contextDocIdsKeyPair() {
```

Comment:
I think it's better to define an attribute in `setup` instead of using a getter since the values won't change.
Something like that:
```javascript
const defaultDocIds = Object.entries(this.context).find(([k,v]) => k.startsWith("default_"));
this.contextDefaultDoc = { field: defaultDocIds[0], ids: defaultDocIds[1] };
```
That way, we do the `find`/`startsWith` one time only instead of doing it every time we read this value

Reply:
Done :partying_face: